### PR TITLE
fix(suite): make account visible if it is not empty anymore

### LINKED ISSUE #3212: "Account does not exist" shown for new Ethereum address used via Metamask
**Describe the bug**
I connected my Trezor to Metamask and selected an empty Ethereum address. Transferred some tokens into the address. Connected to Suite. It shows the new address in the left accounts section (with the correct amount of ETH shown), but the main screen shows "Account does not exist". I can withdraw tokens via the Metamask interface talking to Trezor though.

**Info:**
 - Suite Version: App; 20.11.2
 - Browser: NA
 - OS: 5.9.14-arch1-1
 - Firmware Version: Model-T; 2.3.4

**Steps to reproduce:**
1. Connect Metamask to Trezor
2. Select a new empty Ethereum address
3. Transfer some tokens to this address
4. Open suite and goto Accounts
5. You will see the new address account listed in the left section.
6. Click the new Ethereum address account.

**Expected result:**
The main Account view for the address to show up

**Actual result:**
"Account does not exist" screen.

**Screenshots:**
image

**Notes**
- Tried adding a new account via Suite and it just adds another Ethereum address account, but the previous one is still inaccessible, although visible in the side bar.
- I can withdraw tokens via the Metamask interface talking to Trezor though.

### PR DESCRIPTION
**Pull request by slowbackspace**
tldr; of what happened:
1. when we first create an account (`accountActions.create`) we set its visibility to true only if it is not empty (or it is first account that we always want to show). Otherwise it is set to false.
2. if the account was remembered we don't update its visibility ever again (`accountActions.update` doesn't update this field).

To show account in the accounts menu it is enough if it is set to visible or non empty 
https://github.com/trezor/trezor-suite/blob/b6feaf87bbd0534afbe49e0a18248758b09aaacb/packages/suite/src/components/wallet/AccountsMenu/index.tsx#L190
BUT to be able to select (open) certain account it MUST be set to visible, we are not checking if it is not empty https://github.com/trezor/trezor-suite/blob/c4202dd7e61489a9dbc93c0795451ef12bfcfe17/packages/suite/src/actions/wallet/selectedAccountActions.ts#L148
So you could get a behaviour where you can actually see the account in the menu but after opening it suite tells you it doesn't exist.

This PR adds changing `visible` field inside  account update action that should fix the problem.

fix https://github.com/trezor/trezor-suite/issues/3212

### CODE REVIEW

**Review by szymonlesisz — APPROVED** [review]

**Comment by slowbackspace** [comment]
I wanted to wait for Bach's confirmation that it indeed solved the issue for him, but he haven't reported back. But I am like 98% sure it will, so I am making this PR ready for merge 😁

**Comment by xbach** [comment]
Confirming solved after a movement on the account.
